src/libtorch.cc

```diff
@@ -587,8 +590,49 @@ ModelInstanceState::ValidateTypedSequenceControl(
 }
 
 TRITONSERVER_Error*
-ModelInstanceState::ValidateInputs()
+ModelInstanceState::ValidateInputs(const size_t expected_input_cnt)
 {
+  const torch::jit::Method& method = torch_model_->get_method("forward");
+  const auto& schema = method.function().getSchema();
+  const std::vector<c10::Argument>& arguments = schema.arguments();
+
+  // Currently we only support input of type Dict[str, Tensor] when the
```

Comment:
"Currently, we only support models with a single input of type Dict[str, Tensor]."
The way it is currently written means that pytorch backend assumes input of type Dict[str, Tesnor] if there is a single input in the model.